bundle/wavefront-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,293 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "wavefront.com/v1alpha1",
+          "kind": "WavefrontProxy",
+          "metadata": {
+            "name": "example-wavefrontproxy",
```

Comment:
lets be consistent with naming of collector.

Reply:
Would like to keep this name as is for now to avoid re release of collector container to RH with proxy config name changed.